src/main/java/sirius/kernel/xml/Outcall.java

```diff
@@ -78,6 +81,19 @@ public class Outcall {
     public static final String HEADER_ACCEPT = "Accept";
     public static final String HEADER_ACCEPT_DEFAULT_VALUE = "*/*";
 
+    /**
+     * Date time formatter as per
+     * <a href="https://datatracker.ietf.org/doc/html/rfc9110#section-5.6.7">RFC 9110 section 5.6.7.</a>
+     * <p>
+     * In contrast to {@link DateTimeFormatter#RFC_1123_DATE_TIME}, the day must use two digits.
```

Comment:
also, we not always out put UTC (indicated by the legacy zone id GMT) - 1123 uses the local TZ offset like +0200